src/sqlancer/cockroachdb/CockroachDBProvider.java

```diff
@@ -252,8 +252,9 @@ public void generateDatabase(CockroachDBGlobalState globalState) throws Exceptio
             total--;
         }
 
-        if (globalState.getDbmsSpecificOptions().getTestOracleFactory().size() == 1 && globalState
-                .getDbmsSpecificOptions().getTestOracleFactory().get(0).create(globalState) instanceof CockroachDBCERTOracle) {
+        if (globalState.getDbmsSpecificOptions().getTestOracleFactory().size() == 1
```

Comment:
What will this do if `CERT` and another oracle like `TLP` is used?

Reply:
Actually, I still think checking the number equals to 1 is redundant.

Reply:
```shell
java -jar sqlancer-2.0.0.jar --num-threads 1 --username root --password "" cockroachdb --oracle NoREC,CERT

Exception in thread "main" com.beust.jcommander.ParameterException: Invalid value for --oracle parameter. Allowed values:[NOREC, AGGREGATE, GROUP_BY, HAVING, WHERE, DISTINCT, EXTENDED_WHERE, QUERY_PARTITIONING, CERT]
```

Reply:
The really confusing code is here:
[getTestOracleFactory](https://github.com/sqlancer/sqlancer/blob/main/src/sqlancer/cockroachdb/CockroachDBOptions.java#L122) returns a list of oracles, but the [oracle](https://github.com/sqlancer/sqlancer/blob/main/src/sqlancer/cockroachdb/CockroachDBOptions.java#L33) can only be one.

Not sure why it must return a list of oracles, so I updated the code as you requested.